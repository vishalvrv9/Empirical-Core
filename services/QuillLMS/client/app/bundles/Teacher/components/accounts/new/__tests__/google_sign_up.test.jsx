import React from 'react';
import { shallow } from 'enzyme';

import GoogleSignUp from '../google_sign_up'

describe('GoogleSignUp component', () => {

  it('should render', () => {
    const wrapper = shallow(
        <GoogleSignUp />
    );
    expect(wrapper).toMatchSnapshot();
  });


});
