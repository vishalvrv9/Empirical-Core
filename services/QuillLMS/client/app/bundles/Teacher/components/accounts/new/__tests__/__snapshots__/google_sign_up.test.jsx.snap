// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleSignUp component should render 1`] = `
<a
  className="google-sign-up"
  href="/auth/google_oauth2?prompt=consent"
>
  <img
    alt="google icon"
    src="/images/google_icon.svg"
  />
  <span>
    Sign up with Google
  </span>
</a>
`;
